modules/@apostrophecms/i18n/i18n/en.json

```diff
@@ -188,29 +260,50 @@
   "restoreTypeAffirmativeLabel": "Yes, restore {{ type }}",
   "restoredPrevious": "Restored previously published version.",
   "resumeEditing": "Resume Editing",
+  "retryingSaveDocument": "Retrying save document...",
   "richText": "Rich Text",
   "safeModeActive": "Running in safe mode, not showing raw HTML.",
+  "save": "Save",
+  "saveDraft": "Save Draft",
+  "saveAsDraft": "Save as a draft to publish later.",
+  "saveAsDraftAndNew": "Save as a draft and create a new {{ typeLabel }}.",
+  "saveAsDraftAndPreview": "Save as a draft and preview the {{ typeLabel }}.",
+  "saveDraftAndCreateNew": "Save Draft and Create New",
+  "saveDraftAndPreview": "Save Draft and Preview",
+  "savingDocument": "Saving document...",
+  "searchDocType": "Search {{ type }}",
   "searchLocales": "Search Locales",
+  "searchLocalesPlaceholder": "Search locales...",
+  "select": "Select",
   "selectAll": "Select All",
   "selectContent": "Select Content",
+  "selectedMenuItem": "✓ {{ label }}",
```

Comment:
Should the check really be part of this? Could be done with CSS.

Reply:
Changing this is not part of the PR, it's just being made localizable. However I would note that in a right-to-left language it might be good that we made it part of the string so it can be repositioned when localizing the text.

Reply:
Probably could still be handled in CSS for rtl languages but not sure if we have support for that in core yet. There's probably much more in the UI we'll need to adjust for true rtl support.